doc-content/shared-kie-docs/src/main/asciidoc/KieServer/ref-rest-api-dmn-endpoints.adoc

```diff
@@ -0,0 +1,566 @@
+----
+
+.Example response (XML)
+[source,xml]
+----
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<response type="SUCCESS" msg="OK models successfully retrieved from container 'mykjar-project'">
+    <dmn-model-info-list>
+        <model>
+            <model-namespace>http://www.redhat.com/_c7328033-c355-43cd-b616-0aceef80e52a</model-namespace>
+            <model-name>Traffic Violation</model-name>
+            <model-id>_99</model-id>
+            <decisions>
+                <dmn-decision-info>
+                    <decision-id>_3</decision-id>
+                    <decision-name>TrafficViolationDecisionService</decision-name>
+                </dmn-decision-info>
+            </decisions>
+        </model>
+    </dmn-model-info-list>
+</response>
+----
+
+.Example response (JSON)
+[source,json]
+----
+{
+  "type": "SUCCESS",
+  "msg": "OK models successfully retrieved from container 'mykjar-project'",
+  "result": {
+    "dmn-model-info-list": {
+      "models": [
+        {
+          "model-namespace": "http://www.redhat.com/_c7328033-c355-43cd-b616-0aceef80e52a",
+          "model-name": "Traffic Violation",
+          "model-id": "_99",
+          "decisions": [
+            {
+              "decision-id": "_3",
+              "decision-name": "TrafficViolationDecisionService"
+            }
+          ]
+        }
+      ]
+    }
+  }
+}
+----
+--
+
+Evaluate decisions for a specified container::
++
+--
+`POST /server/containers/{containerId}/dmn`
+
+.Example REST endpoint
+`\http://localhost:8080/kie-server/services/rest/server/containers/mykjar-project/dmn`
+
+.Example curl request
+[source]
+----
+curl -u wbadmin:wbadmin -H "accept: application/json" -H "content-type: application/json" -X POST "http://localhost:8080/kie-server/services/rest/server/containers/mykjar-project/dmn" -d "{ \"model-namespace\" : \"http://www.redhat.com/_c7328033-c355-43cd-b616-0aceef80e52a\", \"model-name\" : \"Traffic Violation\", \"decision-name\" : [ ], \"decision-id\" : [ ], \"dmn-context\" : }"
+----
+
+.Example POST request body with input data
+[source,json]
+----
+{
+  "model-namespace" : "http://www.redhat.com/_c7328033-c355-43cd-b616-0aceef80e52a",
+  "model-name" : "Traffic Violation",
+  "decision-name" : [ ],
+  "decision-id" : [ ],
+  "dmn-context" : [ ],
+}
+----
+
+.Example response (JSON)
+[source,json]
+----
+{
+  "type": "SUCCESS",
+  "msg": "OK from container 'mykjar-project'",
+  "result": {
+    "dmn-evaluation-result": {
+      "messages": [],
+      "model-namespace": "http://www.redhat.com/_c7328033-c355-43cd-b616-0aceef80e52a",
+      "model-name": "Traffic Violation",
+      "decision-name": [],
+      "dmn-context": [],
+      "decision-results": {
+        "_3": {
+          "messages": [],
+          "decision-id": "_3",
+          "decision-name": "TrafficViolationDecisionService",
+          "status": "SUCCEEDED"
+        }
+      }
+    }
+  }
+}
+----
+--
```

Comment:
Thanks for raising this point. Since the legacy endpoints are already available in a different .adoc file, I've dedicated this section for model-specific endpoints and removed the legacy endpoints.